### PR TITLE
README, tools/dm: update wording

### PR DESCRIPTION
**Pull request by CaitinChen**
Related PR: https://github.com/pingcap/docs/pull/887
@lilin90 @csuzhangxc @yikeke PTAL

### CODE REVIEW

File: tools/dm/shard-merge.md [hunk 1]
```diff
@@ -19,22 +19,22 @@ DM has the following sharding DDL usage restrictions:
...
 
 ### Background
 
-Currently, DM uses the binlog in the `ROW` format to perform the replication task. The binlog does not contain the table schema information. When you use the `ROW` binlog to replicate data, if you have not replicated multiple upstream tables into the same downstream table, then there only exists DDL operations of one upstream table that can update the table schema of the downstream table. The `ROW` binlog can be considered to have the nature of self-description. During the replication process, the DML statements can be constructed accordingly with the column values and the downstream table schema.  
+Currently, DM uses the binlog in the `ROW` format to perform the replication task. The binlog does not contain the table schema information. When you use the `ROW` binlog to replicate data, if you have not replicated multiple upstream tables into the same downstream table, then there only exist DDL operations of one upstream table that can update the table schema of the downstream table. During the replication process, the DML statements can be constructed accordingly with the column values and the downstream table schema.
```
Comment (lilin90) [inline tools/dm/shard-merge.md]: Why remove this sentence?

**The `ROW` binlog can be considered to have the nature of self-description.**

File: tools/dm/shard-merge.md [hunk 2]
```diff
@@ -46,29 +46,29 @@ In the above example, the merging process is simplified, where only two MySQL in
...
 
-1. `DM-worker-1` receives the DDL statement from MySQL instance 1 at `t1`, pauses the data replication of the corresponding DDL and DML statements itself, and sends the DDL information to `DM-master`.
-2. `DM-master` decides that the replication of this DDL statement needs to be coordinated based on the received DDL information, creates a lock for this DDL statement, sends the DDL lock information back to `DM-worker-1` and marks `DM-worker-1` as the owner of this lock at the same time.
+1. `DM-worker-1` receives the DDL statement from MySQL instance 1 at `t1`, pauses the data replication of the corresponding DDL and DML statements, and sends the DDL information to `DM-master`.
+2. `DM-master` decides whether to coordinate the replication of this DDL statement based on the received DDL information, creates a lock for this DDL statement, sends the DDL lock information back to `DM-worker-1` and marks `DM-worker-1` as the owner of this lock at the same time.
```
Comment (lilin90) [inline tools/dm/shard-merge.md]: It seems that using `whether` in this example is inappropriate. Same for other similar "whether" below.

File: tools/dm/shard-merge.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ category: tools
...
 
-This document introduces the sharding support feature provided by Data Migration (DM). This feature allows you to merge and replicate the data of tables with the same table schema in the upstream MySQL or MariaDB instances into one same table in the downstream TiDB. It supports not only replicating the upstream DML statements, but also coordinating to replicate the table schema modification using DDL statements in multiple upstream sharded tables.
+This document introduces the sharding support feature provided by Data Migration (DM). This feature allows you to merge and replicate the data of tables with the same table schema in the upstream MySQL or MariaDB instances into one same table in the downstream TiDB. It supports not only replicating the upstream DML statements, but also coordinating to replicate the table schema change using DDL statements in multiple upstream sharded tables.
 
-> **Note:** To merge and replicate data from the sharded tables, you must configure the `is-sharding: true` parameter in the task configuration file.
+> **Note:** To merge and replicate data from the sharded tables, you need to set `is-sharding` to "true" in the task configuration file.
```
[the remark this answers is not in the document]
Comment (GregoryIan) [reply tools/dm/shard-merge.md]: it's right

File: tools/dm/shard-merge.md (hunk 2 above)
Comment (csuzhangxc) [reply tools/dm/shard-merge.md]: Agree. DM-master always coordinates the replication if needed.

File: tools/dm/shard-merge.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lilin90) [reply tools/dm/shard-merge.md]: @GregoryIan Do you mean the version provided by @CaitinChen is right? Please note that when the user edits the config file, this item might not exist.

File: tools/dm/shard-merge.md (hunk 2 above)
Comment (lilin90) [reply tools/dm/shard-merge.md]: @CaitinChen So please update. Thanks!

File: tools/dm/shard-merge.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (CaitinChen) [reply tools/dm/shard-merge.md]: @GregoryIan Please share your idea.

File: tools/dm/shard-merge.md (hunk 1 above)
Comment (CaitinChen) [reply tools/dm/shard-merge.md]: I will add it back.

File: tools/dm/shard-merge.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (GregoryIan) [reply tools/dm/shard-merge.md]: `is-sharding: true` is better 🤔

[the remark this answers is not in the document]
Comment (CaitinChen) [reply tools/dm/shard-merge.md]: Thanks.

**Comment by CaitinChen** [comment]
@csuzhangxc @lilin90 PTAL again

**Comment by csuzhangxc** [comment]
LGTM

File: tools/dm/shard-merge.md [hunk 4]
```diff
@@ -86,17 +86,17 @@ Assume that there are two sharded tables, namely `table_1` and `table_2`, to be
...
 4. At `t3`, the Syncer unit in DM-worker receives the DDL statements of `table_2`.
 5. From `t4` on, the Syncer unit in DM-worker receives the DML statements of `schema V2` from both tables.
 
-If we do not handle the DDL statements particularly during the data synchronization, when the DDL statement of `table_1` is synchronized to the downstream and alters the downstream table schema, the DML statement of `schema V1` from `table_2` cannot be replicated successfully. Therefore, within a single DM-worker, we also create a logical sharding group similar to that within `DM-master`, except that members of this group are the different sharded tables in the same upstream MySQL instance.
+If the DDL statements are not processed particularly during the data synchronization, when the DDL statement of `table_1` is replicated to the downstream and changes the downstream table schema, the DML statement of `schema V1` from `table_2` cannot be replicated successfully. Therefore, within a single DM-worker, a logical sharding group similar to that within `DM-master` is created, except that members of this group are different sharded tables in the same upstream MySQL instance.
```
Comment (lilin90) [inline tools/dm/shard-merge.md]:
```suggestion
If you do not handle the DDL statements particularly during the data replication, when the DDL statement of `table_1` is replicated to the downstream and changes the downstream table schema, the DML statement of `schema V1` from `table_2` cannot be replicated successfully. Therefore, within a single DM-worker, a logical sharding group similar to that within `DM-master` is created, except that members of this group are different sharded tables in the same upstream MySQL instance.
```

**Comment by CaitinChen** [comment]
@lilin90 @csuzhangxc PTAL again

**Pull request merged**